### PR TITLE
test: fix locale e2e test

### PR DESCRIPTION
**Pull request by shairez**
# Overview
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Fixes the broken locale e2e test
# Use cases and why

### CODE REVIEW

File: starters/e2e/qwikcity/locale.e2e.ts [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
 test.describe("Qwik City locale API", () => {
-  // TODO(v2): fix this
-  test.skip("pass locale to Qwik", async ({ page }) => {
+  test("pass locale to Qwik", async ({ page }) => {
     await page.goto("/qwikcity-test/locale");
     const locale = page.locator(".locale");
-    const qContainer = page.locator("[q\\:container]");
     await expect(locale).toHaveText("test-locale");
+    const qContainer = page.locator("[q\\:container=paused]");
```
Comment (Varixo) [inline starters/e2e/qwikcity/locale.e2e.ts:8]: yeah this is a good way to solve this, but I think we should use the same selector as the rest of the system
Look here https://github.com/QwikDev/qwik/blob/3feb2e6da2e4e9725dd8331e2730ba13e88e63d5/packages/qwik/src/core/util/markers.ts#L42

**Comment by Varixo** [comment]
Could we skip `Cloudflare Pages` for v2 branch? Is it a good idea?

File: starters/e2e/qwikcity/locale.e2e.ts (hunk 1 above)
Comment (shairez) [reply starters/e2e/qwikcity/locale.e2e.ts:8]: good idea!

**Comment by shairez** [comment]
@mhevery do you think we need the cloudflare pages on the v2 branch?

for some reason it's failing on not finding the CLI

File: starters/e2e/qwikcity/locale.e2e.ts (hunk 1 above)
Comment (shairez) [reply starters/e2e/qwikcity/locale.e2e.ts]: done

**Comment by Varixo** [comment]
@shairez can we merge this?

**Comment by shairez** [comment]
Yeah I'll do it

**Pull request merged**